### DISCUSSION TITLE
Support for an endpoint [get_current_tweaks_by_flow_id]?

### DISCUSSION THREAD

**Discussion by zdql** [langflow-ai/langflow#2486] [Ideas]
Hi all! New to the repo, but wondering if a backend endpoint for getting the current tweaks from a flow-id would be possible.

The current configuration of tweaks makes it difficult to control / invoke flows from a backend endpoint.

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
